pingdom/resource_pingdom_check.go

```diff
@@ -504,6 +504,11 @@ func resourcePingdomCheckRead(d *schema.ResourceData, meta interface{}) error {
 	}
 	d.Set("teamids", teamids)
 
+	if probefilters := ck.ProbeFilters; len(probefilters) > 0 {
+		// normalise: "region: NA" -> "region:NA"
+		d.Set("probefilters", strings.Replace(probefilters[0], ": ", ":", 1))
```

Comment:
Nice catch!  I'll admit the Pingdom API is a bit inconsistent here which makes this a little cumbersome to deal with.

The API allows for comma delimited strings I can set something like this in the terraform plan which will set multiple regions:

```
resource "pingdom_check" "example" {
  type       = "http"
  name       = "test check"
  host       = "example.com"
  resolution = 5
  probefilters = "region:NA, region: EU"
}
```

If I read back the check using the API I can see both regions are set.  However in the Pingdom webapp, under `Edit Check` the regions selection uses radio buttons which makes it appear as if only one item can be selected:

<img width="657" alt="pingdom-probe-filter-selection" src="https://user-images.githubusercontent.com/768346/69126209-8140d480-0a5c-11ea-842f-4d5cde6a1276.png">

With your change here we at least get back the first item which is great, but it looks like we could set multiple regions through the API only and as you mention, this change would only see the first item which makes the plan look out of date.

I tried something like this here instead which seems to do a bit better:
```
d.Set("probefilters", strings.ReplaceAll(strings.Join(probefilters, ", "), ": ", ":"))
```

Although there'd still be problems with spacing, for instance if someone used `probefilters = "region:NA,region: EU"` in their plan instead of `probefilters = "region:NA, region: EU"`.

Maybe the way forward here is:

* Allow for comma delimited strings on read and write for now
* We can make the breaking change as part of a 2.0.0 release (we'd probably want to think about other breaking changes we need to make too)

On that last one, Pingdom also moved to a 3.X API which we don't support yet.  I'm not sure right now if that introduces other breaking changes but maybe as part of migrating to that we can also fix the probefilters type to be a set of strings.  Any thoughts on this?